test/base.py

```diff
@@ -45,33 +48,36 @@ def flatten(l, ltypes=(list, tuple)):
         i += 1
     return ltype(l)
 
+
 class ColumnName(str):
     pass
 
+
 def dbparam(x):
     if x is None:
         return 'NULL'
     elif isinstance(x, ColumnName):
         return "`" + x + "`"
-    elif isinstance(x, basestring):
+    elif isinstance(x, bytes):
+        return "'" + str(x) + "'"
```

Comment:
I think this needs to be done as:
```python
    return "'" + str(x.decode()) + "'"
```
to decode the bytes. Otherwise the `b` turns up in the string.